### ISSUE TITLE
Bug: Export is in incorrect format for SurrealDB `2.x`

### ISSUE THREAD

**Issue by tobiemh** [surrealdb/surrealist#474]
### Describe the bug

When performing an export from within Surrealist, the export format is still using the SurrealDB `1.x` format.

In addition, and as a result, there can be issues with importing this format into SurrealDB `2.x` as certain things need to be handled and quoted correctly for the new parser. As a result, some imports fail, even though Surrealist suggests that the import succeeded without any issues.

### Steps to reproduce

1. Start a clean database in Surrealist and execute the following query:

```sql
USE NS test DB test;
CREATE person:test;
```

2. Perform an export in Surrealist. The following SurrealQL will be exported:

```sql
-- ------------------------------
-- Export generated by Surrealist on 2024-09-25T19:09:43.576Z
-- ------------------------------

OPTION IMPORT;

-- ------------------------------
-- TABLE: person
-- ------------------------------

DEFINE TABLE person TYPE ANY SCHEMALESS PERMISSIONS NONE;

-- ------------------------------
-- TRANSACTION
-- ------------------------------

BEGIN TRANSACTION;

-- ------------------------------
-- TABLE DATA: person
-- ------------------------------

UPDATE person:test CONTENT { id: person:test };

-- ------------------------------
-- TRANSACTION
-- ------------------------------

COMMIT TRANSACTION;
```

4. Perform an export using the command-line tool. The following SurrealQL will be exported:

```sql
-- ------------------------------
-- OPTION
-- ------------------------------

OPTION IMPORT;

-- ------------------------------
-- TABLE: person
-- ------------------------------

DEFINE TABLE person TYPE ANY SCHEMALESS PERMISSIONS NONE;

-- ------------------------------
-- TABLE DATA: person
-- ------------------------------

INSERT [ { id: person:test } ];
```

### Expected behaviour

Should export according to the SurrealDB `2.x` export format, not the SurrealDB `1.x` format, and should ensure that anything which needs to be quoted is done so correctly.

### Surrealist Environment

Version: 3.0.2
Flags: feature_flags: true, query_view: true, explorer_view: true, graphql_view: true, designer_view: true, auth_view: true, functions_view: true, models_view: true, apidocs_view: true, cloud_view: true, themes: true, newsfeed: true, database_version_check: false, highlight_tool: false, legacy_serve: false, cloud_endpoints: production, cloud_access: true, cloud_killswitch: true, changelog: auto

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tobiemh** [comment]
Closing this issue, may also close:
- https://github.com/surrealdb/surrealdb/issues/4691
- https://github.com/surrealdb/surrealdb/issues/4430

**Comment by macjuul** [comment]
Fixed in #477
